tripy/tests/backend/mlir/test_compiler.py

```diff
@@ -53,9 +53,9 @@ def test_reason_context(self):
         with FlatIRTensor.context(["This is the first level of context"]):
             with FlatIRTensor.context(["This is the second level of context"]):
                 # We need to emit an error from one of the internally created `FlatIRTensor`s to see the context
-                a = tp.ones(1)
-                b = tp.ones(1)
```

Comment:
Is this intended to work? The signatures for `full`, `ones`, etc., indicate not, but `np.ones`, `torch.ones`, etc., would treat this as equivalent to `ones((1,))`.

Reply:
Not intended to work. `np.ones` and `torch.ones` accept variadic positional args which is why they would treat this as `(1,)`, while we do not currently.